enzyme/Enzyme/MLIR/Dialect/EnzymeOps.td

```diff
@@ -132,4 +134,15 @@ def ShadowedGradient : Enzyme_Type<"ShadowedGradient"> {
   let assemblyFormat = "`<` $basetype `>`";
 }
 
+def AddToOp : Enzyme_Op<"addTo">, //, [Pure, Terminator, ReturnLike]
```

Comment:
@wsmoses Is it possible to have this inherit from linalg.yield to pass the verifier of linalg.generic? Also any idea for a better name?

Reply:
Not that I'm aware of unfortunately.

Reply:
@ftynse may have more info tho

Reply:
What should I use if thats not possible? Just make a custom linalg generic op?